src/main/java/org/isf/serviceprinting/manager/PrintManager.java

```diff
@@ -116,4 +116,4 @@ public void print(String filename, List<?> toPrint, int action) throws OHService
 			LOGGER.error(jrException.getMessage(), jrException);
 		}
 	}
-}
+}
```

Comment:
You will find that about half the code ends in a new line and the other half doesn't.   Making this kind of change is self-defeating as the next person will probably change it back.